h2o-hadoop/scripts/docker/cdh/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM nimmis/java:14.04-oracle-8-jdk
+
+ARG VERSION
+ARG PATH_PREFIX='.'
+
+ENV DISTRIBUTION 'cdh'
+ENV VERSION $VERSION
+ENV SETUP_H2O TRUE
+ENV START_H2O TRUE
+ENV RUN_TESTS TRUE
+ENV ENTER_BASH FALSE
+
+RUN echo "# Packages for Cloudera's Distribution for Hadoop, Version 5, on Ubuntu 14.04 amd64\n\
+deb [arch=amd64] http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh trusty-cdh${VERSION} contrib\n\
+deb-src http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh trusty-cdh${VERSION} contrib\n" > /etc/apt/sources.list.d/cloudera.list
+
+# Prepare Cloudera repository
+COPY ${PATH_PREFIX}/conf/cloudera.pref /etc/apt/preferences.d/cloudera.pref
+RUN wget http://archive.cloudera.com/cdh5/ubuntu/trusty/amd64/cdh/archive.key -O archive.key && sudo apt-key add archive.key && \
+    sudo apt-get update
+
+# Download necessary packages
+RUN apt-get install -y hadoop-conf-pseudo python-dev python-pip python-scipy
+RUN pip install requests --upgrade
+RUN pip install colorama future tabulate pandas
```

Comment:
This is probably good for now, however we need to consider that the list of dependencies can change in the future and we will need to make changes here as well.